tests/utils/runner/network_manager.go

```diff
@@ -66,12 +66,14 @@ func NewDefaultANRConfig() ANRConfig {
 		AvalancheGoExecPath: os.ExpandEnv("$GOPATH/src/github.com/ava-labs/avalanchego/build/avalanchego"),
 		PluginDir:           os.ExpandEnv("$GOPATH/src/github.com/ava-labs/avalanchego/build/plugins"),
 		GlobalNodeConfig: `{
+			"log-level":"info",
 			"log-display-level":"info",
 			"proposervm-use-current-height":true
 		}`,
 		GlobalCChainConfig: `{
 			"warp-api-enabled": true,
-			"log-level": "debug"
+			"log-level": "debug",
+			"log-display-level": "debug"
```

Comment:
```suggestion
			"log-level": "debug",
```

There is no `log-display-level` for the C-Chain specific config

Reply:
removed